### PR TITLE
RAML 1.0 - Fixes example render, Support to render multiple examples

### PR DESCRIPTION
**Pull request by edentsai**
References to the PR #210, 
fixes example redner error, and check media types for JSON or YAML

The commits:
- Render example object to the YAML type by default.
- Render example object of application/json to the JSON type
- Supports to render multiple examples with navigation tabs

See the example.html
- https://rawgit.com/edentsai/raml2html/628b3696b2cee26e16b15c4e6ed7432159336959/examples/raml1.0/example.html

### CODE REVIEW

**Comment by edentsai** [comment]
# Demo - Render example object:
raml2html-examlpe-type

**Comment by edentsai** [comment]
# Demo Render examples 
raml2html-examples

**Comment by sichvoge** [comment]
Nice. Works like a charm so far. Thanks for contributing! 👍

**Comment by jhen0409** [comment]
@edentsai nice! :+1:

File: lib/resource.nunjucks [hunk 1]
```diff
@@ -231,6 +260,35 @@
                             <p><strong>Example</strong>:</p>
                             <pre><code>{{ stringify(b.example, b.name) | escape }}</code></pre>
                           {% endif %}
+
+                          {% if b.examples %}
+                            <p><strong>Examples</strong>:</p>
+                            <ul class="nav nav-tabs">
+                              {% for example in b.examples %}
+                                <li>
```
Comment (edentsai) [inline lib/resource.nunjucks]: Oops, I wrote a redundant `<li>` tag, removes it in #242

File: lib/resource.nunjucks [hunk 2]
```diff
@@ -168,6 +168,35 @@
                           <p><strong>Example</strong>:</p>
                           <pre><code>{{ stringify(b.example, b.name) | escape }}</code></pre>
                         {% endif %}
+
+                        {% if b.examples %}
+                          <p><strong>Examples</strong>:</p>
+                          <ul class="nav nav-tabs">
+                            {% for example in b.examples %}
+                              <li>
```
Comment (edentsai) [inline lib/resource.nunjucks]: Oops, I wrote this redundant `<li>` tag, removes it in #242

**Pull request merged**